contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,365 @@
+
+If you are a test maintainer, refrain from including markdown in strings that
+are used to name your tests and test components.
+
+#### Fill out the issue for a Flaking Test
+
+1 **Which jobs are flaking**
+
+The example below was taken from the SIG Release dashboard:
+
+<img src="./testgrid-images/testgrid-jobs.png" height="50%" width="100%">
+
+We can see that the following jobs were flaky at the time this screenshot was taken:
+
+- [conformance-ga-only]
+- [skew-cluster-latest-kubectl-stable1-gce]
+- [gci-gce-ingress]
+- [kind-master-parallel]
+
+1. **Which tests are flaking**
+
+Let's grab an example from the SIG release dashboards and look at the
+`node-kubelet-features-master` job in sig-release-master-informing [node-kubelet-features-master].
+
+<img src="./testgrid-images/failed-tests.png" height="70%" width="100%">
+
+Here we see that at 16.07 EDT and 15:07 EDT the tests
+
+```text
+[k8s.io] NodeProblemDetector [NodeFeature:NodeProblemDetector]
+[k8s.io] SystemLogMonitor should generate node condition and events for corresponding errors [ubuntu]
+```
+
+Failed for Kubernetes commit `9af86e8db` (3)
+The corresponding test-infra commit was `11cb57d36`
+
+At 15:07 EDT, the test
+
+```text
+[k8s.io] NodeProblemDetector [NodeFeature:NodeProblemDetector]
+[k8s.io] SystemLogMonitor should generate node condition and events for corresponding errors [cos-stable2]
+```
+
+failed as well.
+
+3. **Since when has it been flaking**
+
+You can get the start time of a flake from the header of the TestGrid page
+showing you all the tests. The red numbers in the screen shot above annotate the
+grid headings.
+
+They are:
+
+- 1 This row has the times each Prow job was started, each column on the grid
+  represents a single run of the Prow job
+- 2 This row is the Prow job run id number
+- 3 This is the kubernetes/kubernetes commit id that was tested
+- 4 Theses are the kubernetes/test-infra commit ids that were used to build and
+run the Prow Job; kubernetes/test-infra contains CI job definition yaml, builds
+for container images used in CI on the Kubernetes project, and also code that
+implements a lot of the components used to deliver CI, such as Prow, SpyGlass
+and other components.
+
+Click on a cell in the grid to take you to SpyGlass which displays the Prow job
+results.
+
+You can also find this data in Triage (see below).
+
+4. **Reason for failure**
+
+Logging an issue brings the flake or failure to the attention of the wider
+community, as the issue reporter you do not have to find the reason for failure
+right away (nor the solution). You can just log the error reported by the test
+when the job was run.
+
+Click on the failed runs (the red cells in the grid)  to see the results in 
+SpyGlass.
+
+For `node-kubelet-features-master`, we see the following:
+
+![Spyglass Prow Job Results for node-kubelet-features-master`](./testgrid-images/spyglass-summary.png "Spyglass Prow Job results viewer")
```

Comment:
nit: is there a reason why this image (and the next) are included with Markdown notation whereas the ones before used HTML? this should be consistent